### DISCUSSION TITLE
我设置好了隧道，但是没有分配二级域名

### DISCUSSION THREAD

**Discussion by abelieve** [natfrp/wiki#300] [使用问题]
![image](https://user-images.githubusercontent.com/20368214/229405611-15e72192-e5be-4e19-841e-c3187c94cf6c.png)

**Comment by LTHPKBTE** [comment]
这是新域名格式，用来应对部分地区的DNS污染。旧格式和新格式以及 IP 应该都能正常使用。

**Reply by wh954** [reply]
请问旧的域名格式要怎么查询啊？新的ip我有时候访问不了，特别是使用vscode连服务器的时候

**Reply by LTHPKBTE** [reply]
旧域名格式已不再使用，部分节点不存在旧格式。IP不能访问的情况下域名一样不能访问的，不存在效果问题，不过节点IP改变时域名会自动更改，IP需要手动更改而已

**Reply by LTHPKBTE** [reply]
如果你无法通过域名访问，优先检查dns
